### ISSUE TITLE
iOS product video media requires extra tap to play

### ISSUE THREAD

**Issue by klawrz89** [Shopify/dawn#1362]
**Describe the current behavior**

- TLDR: When viewing product video media on iOS devices, 3 taps are required to get the video to play (as opposed to 2 taps on Android)

- Full description: When viewing product media on iOS, video media does not play until you tap the play button in the *actual device video player controls* — this means you need to a) tap the media thumbnail to focus on the video, b) tap the 'play'  button in the center of the video, AND c) tap the 'play' button in the video player controls to actually start the video

**Describe the expected behavior**

- Users should be able to start a video by a) tapping the media thumbnail to give focus to the video, b) tap the centered play button to start the video. The video should play at this point, without the user needing to tap the play button in the controls

**Version information (Dawn, browsers and operating systems)**
- Dawn Version: 3.0.0
- Chrome Version 98.0.4758.80 (Official Build) (x86_64)
- iOS v15.2 on iPhone 13

**Possible solution**

- Perhaps an autoplay attribute could be used to force play after 2nd click

**Additional context/screenshots**

- Behaviour in iOS (3 taps, final tap on video player controls necessary to start video):
![https://screenshot.click/22-02-al2br-ppy14.gif](https://screenshot.click/22-02-al2br-ppy14.gif)

- Behaviour on Android (only 2 taps, no interaction with player controls necessary to start video):
![https://screenshot.click/22-02-k6btn-wqp1o.gif](https://screenshot.click/22-02-k6btn-wqp1o.gif)
